.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,21 @@
 <!-- PR Guidelines - Inspired by https://jesseduffield.com/Submitting-PRs/
 
-* Read your PR thoroughly as if you were the reviewer.
-* Add reviewers ONLY WHEN the PR is ready for general review (i.e. not a Draft)
-* If you seek feedback before general review, start a Draft PR and add specific
-  reviewers. Note: CODEOWNERS are automatically assigned, cannot be removed and
-  are instructed to disregard Draft PRs.
-* Squash commits before review. Combine your changes into a clear and concise
-  set of commits.
-* Avoid force pushing new commits once reviewers are assigned as this makes
-  tracking changes difficult. If required, the change can be squashed again
-  after approval.
+* SELF-REVIEW YOUR PR: Read your PR thoroughly as if you were the reviewer.
+
+* ADD REVIEWERS ONLY WHEN READY: Add reviewers only when the PR is ready (not
```

Comment:
Can probably remove the text in parentheses as the next point addresses Draft specifically.